packages/swing-store/src/bundleStore.js

```diff
@@ -72,6 +72,7 @@ export function makeBundleStore(db, ensureTxn, noteExport = () => {}) {
    * @param {Bundle} bundle
    */
   function addBundle(bundleID, bundle) {
+    bundleID.startsWith('b1-') || Fail`invalid bundleID ${q(bundleID)}`;
```

Comment:
This will collide with my #7242, whichever one lands last will need to resolve a merge conflict.

Reply:
I think we're both very close to landing, so the question is probably which order will make the conflict resolution easiest.